packages/ui-components/src/Button/Button.tsx

```diff
@@ -2,12 +2,29 @@ import React, { FC } from 'react';
 
 type Props = {
   title?: string;
+  buttonColor: string;
+  textColor?: string;
+  disabled?: boolean;
+  block?: boolean;
+  rounded?: boolean;
+  size?: string;
   onClick?: React.MouseEventHandler<HTMLButtonElement>;
 };
 
-const Button: FC<Props> = ({ onClick, title }) => (
-  <button onClick={onClick} className={'bg-red-800 border-gray-800 border'}>
-    {title && <span className="w-full p-3">{title}</span>}
+const Button: FC<Props> = ({ title, buttonColor, textColor, disabled = false, block, rounded, size, onClick}) => (
+  <button 
+    onClick={onClick} 
+    className={
+      `${buttonColor && !disabled ? 'bg-'+buttonColor : 'primary'} 
+      text-${textColor ? textColor : 'white'}
+      ${rounded && 'rounded-full'}
+      ${block && 'w-full'} 
```

Comment:
when rounder (same applies for block) is falsy, this button will have 'undefined'\'false'\'null' class in className prop
you can use rounded?'rounded-full':'' or create a function:
(flag, className)=>flag?className:''
and use it wherever needed to avoid undefined in class names